spec/routing/uploads_spec.rb

```diff
@@ -0,0 +1,79 @@
+require "spec_helper"
+
+describe "the uploads" do
+  include Capybara::DSL
+
+  context "user not logged in" do
+    it "visits uploads_path" do
+      visit uploads_path
+
+      expect(current_path).to eq(root_path)
+    end
+
+    it "visits new_upload_path" do
+      visit new_upload_path
+
+      expect(current_path).to eq(root_path)
+    end
+
+    it "visits edit_upload_path" do
+      upload = create(:upload)
+
+      visit edit_upload_path(id: upload.id)
+
+      expect(current_path).to eq(root_path)
+    end
+  end
+
+  context "user logged in" do
+    before { user = create(:user)
+      allow_any_instance_of(ApplicationController).
+      to receive(:current_user).and_return(user) }
```

Comment:
Expression at 31, 50 should be on its own line.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.